### PR TITLE
FEAT: Added images and data to show from mockbee.

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, 
Things to check:

- [x] Code for product-page-body.jsx.
- [x] Importing of images. 

What not to check
- [ ]  The functionality of the app
- [ ] Buttons
- [ ] navbar
- [ ] Images

[Preview link](https://deploy-preview-9--need-games-zoef.netlify.app/products) **Click on  "Find More Products"**

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
1.Find new way of importing image struggling with it  
2. you can use Context for data passing may be you will implement after sometime 
All looks good to merge .

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and the UI is awesome
Looks good to me

File: src/assets/images/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	NintendoSwitchJoycon2: require('./accessories/NintendoSwitchJoycon-2.png').default,
+	PS5ControllerB1: require('./accessories/ps5controller-b-1.png').default,
+	PS5ControllerB2: require('./accessories/ps5controller-b-2.png').default,
+	PS5ControllerR1: require('./accessories/ps5controller-r-1.png').default,
+	PS5ControllerR2: require('./accessories/ps5controller-r-2.png').default,
+	XboxController1: require('./accessories/xbox-controller-1.jpg').default,
+	XboxController2: require('./accessories/xbox-controller-2.png').default,
+};
+
+export default IMAGES;
```
Comment (iRohitGaur) [inline src/assets/images/index.js:30]: nice way to import images

**Review by iRohitGaur — APPROVED** [review]
UI looks great. Good to merge 👍🏻

**Pull request merged**